packages/datagrid/src/components/DefaultCellRenderer/DefaultValueRenderer.component.js

```diff
@@ -51,7 +51,7 @@ export default class DefaultValueRenderer extends React.Component {
 	render() {
 		let stringValue;
 
-		if (this.props.value === null) {
+		if (this.props.value === null || this.props.value === undefined) {
```

Comment:
Is this should be enought ?
```suggestion
		if (this.props.value == null) {
```

Reply:
exact, with abstract equality undefined == null && null == null. But this rule is often bad understand. So, i checked this to be sure https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness#Loose_equality_using ;)

Reply:
Alright, i don't care that much on this point ;)